third_party/terraform/website/docs/r/bigtable_instance.html.markdown

```diff
@@ -86,7 +86,9 @@ for a `DEVELOPMENT` instance.
 
 ## Attributes Reference
 
-Only the arguments listed above are exposed as attributes.
+* id - an identifier for the resource with format projects/{{project}}/instances/{{name}}
+
+* name - The unique name of the requested big table instance. Values are of the form projects/<project>/instances/<instanceId>.
```

Comment:
This isn't an exported field on this resource. It's a user-configured field. https://github.com/GoogleCloudPlatform/magic-modules/pull/3207/files#diff-cabec47f1c3393e21fdded661c4b35a8R51

Unlike the other resource we were talking about, we use a field called `name` to supply what the API calls the instanceId.

Reply:
removed the field